src/stravalib/client.py

```diff
@@ -731,7 +735,7 @@ def create_activity(
         activity_type: ActivityType | None = None,
         description: str | None = None,
         distance: pint.Quantity | float | None = None,
-    ) -> model.Activity:
+    ) -> model.DetailedActivity:
```

Comment:
it's unlikely that create_activity will need Detailed level of attrs... double check.